.azure-pipelines/azure-pipelines-external.yml

```diff
@@ -10,11 +10,12 @@ jobs:
   timeoutInMinutes: 360
   strategy:
     matrix:
-      Python_37_Latest:
+      Python_37_Latest_PyMC3_gh:
```

Comment:
Why gh can't be in special group?

Reply:
it can, I just set it in the first group because the requirement we list is github version (and for now we only tested on gh version) so I assumed it was the "official" dependency (first group) and the latest release was the extra check (group below). 

It will be cleaner the other way round as you say though. I'll wait to check everything works (to be sure no extra changes are needed) before pushing so I don't cancel the run